cql3/selection/selection.cc

```diff
@@ -454,7 +453,7 @@ std::vector<const column_definition*> selection::wildcard_columns(schema_ptr sch
     return
         columns |
         std::views::filter([](const column_definition& c) {
-            return !c.is_hidden_from_cql();
+            return !c.is_view_virtual();
```

Comment:
Why don't you need to exclude the "internal" columns here?
I thought you need to do that on wildcard.

Reply:
Oh, I see you fixed that in the next patch. Ok.